Chapter4/proRing.erl

```diff
@@ -0,0 +1,18 @@
+-module(proRing).
+-export([ring/2]).
+
+%%Exercise 4-2: The Process Ring
+%Not sure what was meant by 'number of messages'
```

Comment:
Idea of this exercise (as far as I can tell) is to create a ring of N processes, that once ring is created, controller (or "first" process in the ring, depends on design) starts sending M messages to next connected processes. Messages are passed to neighbouring processes until they reaches originator.